### ISSUE TITLE
Feature request to select previous dates as well

### ISSUE THREAD

**Issue by ravensnowbird** [liloow/vue-cal#5]
I really love this calendar. I'm using this for events calendar but currently it only supports ( Today + Future ). So there's no way for a user to edit / select a previous date event.  What's the best way to enable this ?

**Comment by ravensnowbird** [comment]
I tried to give it a shot myself. When i run `npm run serve` i get the following output
```
 DONE  Compiled successfully in 1727ms                                                                                                                         18:20:35

 
  App running at:
  - Local:   http://localhost:8080/ 
  - Network: http://192.168.1.10:8080/

  Note that the development build is not optimized.
  To create a production build, run npm run build.
```
But when i open the url, the page is empty. Any help would be great.

**Comment by liloow** [comment]
First of all, I do apologize for the delay in my response! I just got a new job and the repo is on Gitlab and I completely neglected Github!

It's a nice idea, thanks for the suggestion! And to make amends for my lack of response, I am pleased to tell you that I implemented this as of version `2.5.0` (or the latest one at the time of writing) ref: #6 

I'll let you check it out and see if it corresponds to what you had in mind! Please don't hesitate to tell me if I missed anything!

**Comment by liloow** [comment]
> I tried to give it a shot myself. When i run `npm run serve` i get the following output
> 
> ```
>  DONE  Compiled successfully in 1727ms                                                                                                                         18:20:35
> 
>  
>   App running at:
>   - Local:   http://localhost:8080/ 
>   - Network: http://192.168.1.10:8080/
> 
>   Note that the development build is not optimized.
>   To create a production build, run npm run build.
> ```
> But when i open the url, the page is empty. Any help would be great.

And that's my bad, I really poorly chose where to describe the right approach (i.e. right above the screenshot) 

It is now fixed and working with the `package.json` scripts and I added clearer instructions to the `README`. It should work as expected from now on!
